web/components/admin/ApplicantInfo/index.js

```diff
@@ -36,13 +44,17 @@ class ApplicantInfo extends React.Component {
 
   render() {
     const { className } = this.props;
+    const shortInfo = this.getApplicantShortInfo();
+    const longInfo = this.getApplicantLongInfo();
     return (
       <div className={`applicant-info pad-sides-s pad-ends-s ${className}`}>
-        <i>TODO: Applicant info component</i>
```

Comment:
Since we've only implemented ApplicantInfo for hacker so far, I think it would be best to show this TODO line if `applicantType` is `mentor` or `volunteer`, and show the fields otherwise.

Reply:
will do